README.md

```diff
@@ -1,13 +1,12 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
-To Do list: list structure
+## To Do list: interactive list
 
 > Description the project.
 
-In this project, I built a simple HTML list of To Do tasks. 
-The list was styled according to the specifications listed later in this lesson. 
-This simple web page was built using webpack and served by a webpack dev server.
+In this project, 
+I added some functionality to my application to make it interactive. The user will also be able to mark task completion by selecting the corresponding checkbox (or undo it by unchecking the checkbox). The updated tasks list will be stored in local storage.
 
 
```

Comment:
- [ ] Kindly complete the getting started section on how the user can clone and run the project locally.